### ISSUE TITLE
Tooltips behave identical when Control mouse filter is set to "Pass" or "Stop"

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot#55430]
### Godot version

3.4.1.rc1

### System information

Win 64, Nvidea 765M

### Issue description

"Stop" and "Ignore" behave exactly like you would expect: The top Control nodes tooltip shows up everwhere if it has "stop" mouse filter, and it does not show up if mouse filter is set to "ignore".

However "Pass" currently just mimics "Spot" behaviour.

I would expect the tooltip from the Control below to show up when the top Control has the "pass" mouse filter and I hover with my mouse over both. If the top Control has "Pass" mouse filter, it should only show up in the region where there is no other Control tooltip below it.

Green Control (top) has set mouse filter to "Pass", but behaves like "Stop":

https://user-images.githubusercontent.com/47016402/143816292-3916572a-abbd-44ce-af13-aa9132b37a7f.mp4

In regions where there is a Control with tooltip below it, imho either both tooltips should show up or just the bottom one.
possible similarities to #54529

### Steps to reproduce

1. Add two Control Nodes, and set a Tooltip for both. Make sure the Controls rects overlap.
2. Set the mouse filter of the top Control to "Pass".
3. Observe how the top Control actually stops the bottom Tooltip from appearing.

**Comment by KoBeWi** [comment]
I think this makes sense. Tooltip shows for the top-most control visually, not input-wise. It would be unexpected if the tooltip showed for a control under the one you are hovering.

**Comment by golddotasksquestions** [comment]
> It would be unexpected if the tooltip showed for a control under the one you are hovering.

yes. When the top Contol has mouse filter "Stop". However when the top node has mouse filter "Pass" I would expect to either see both tooltips or just the one from the bottom Control as is it the one closer to the root.
